src/components/searchbar/Searchbar.css

```
@@ -0,0 +1,32 @@
+.searchbar {
+    display: flex;
+    justify-self: flex-start;
+    align-items: center;
+    background-color: var(--bg-secondary);
+    border-radius: 15px;
+    padding: 10px 10px 10px 32px;
+    margin-top: 36px;
+}
+
+.searchbar-input {
+    width: 100%;
+    font-size: 1.125rem;
+    color: var(--text-color);
+    background: none;
+    margin-left: 24px;
+    border: none;
+    outline: none;
+    height: 48px;
```

Group together CSS properties with related functions
```suggestion
    width: 100%;
    height: 48px;
    font-size: 1.125rem;
    color: var(--text-color);
    background: none;
    margin-left: 24px;
    border: none;
    outline: none;
```